tests/functional/pages/surveys/relationship/household-relationship.page.js

```diff
@@ -9,12 +9,14 @@ class HouseholdRelationshipPage extends QuestionPage {
 
   setHusbandOrWifeRelationship(index, relationshipIndex) {
     var id = this.buildRelationshipAnswerId(index, 1)
+    browser.waitForExist(id)
```

Comment:
I'll be surprised if this works fixes the intermittent issue as the element does exist as it is trying to click it when it fails. I think the problem is that the animation changes the location of the elements from the location where the driver first detects the element is located